app/models/agents/dropbox_file_url_agent.rb

```diff
@@ -32,14 +33,42 @@ class DropboxFileUrlAgent < Agent
 
     MD
 
-    event_description <<-MD
-      The event payload will contain the following fields:
-
+    event_description do
+      "Events will looks like this:\n\n    %s" % if options['link_type'] == 'permanent'
+        Utils.pretty_print({
+          :url=>"https://www.dropbox.com/s/abcde3/example?dl=1",
```

Comment:
I think new code should adopt the new style hash notation.  `{ url: ..., ".tag": ... }`

Reply:
True, thanks! I was spending too much time figuring out how to get the new omniauth plugin working without changing the callback url and god lazy 😊